libnmstate/schema.py

```diff
@@ -437,3 +439,14 @@ class Mode:
 class MacVtap(MacVlan):
     TYPE = InterfaceType.MAC_VTAP
     CONFIG_SUBTREE = "mac-vtap"
+
+class Wireguard:
+    TYPE = InterfaceType.WIREGUARD
+    CONFIG_SUBTREE = "wireguard"
+    FWMARK = "fwmark" 
+    IPV4_AUTO_DEFAULT_ROUTE = "ipv4-auto-default-route"
+    IPV6_AUTO_DEFAULT_ROUTE = "ipv6-auto-default-route"	 
```

Comment:
These two are just a wrapper done by NetworkManager.
The wireguard natively does not have them.

IMHO, user can manually get the same result using `fwmark` and `route rule` as mentioned in https://www.wireguard.com/netns/#routing-all-your-traffic

Please list the use cases which these options are required mandatory.